examples/barotropic_instability.jl

```diff
@@ -0,0 +1,288 @@
+# # Barotropic instability in a channel on a β-plane
+#
+# This example calculates the growth rate of the "barotropic" (that is
+# vertically-uniform) instability of a boundary-hugging jet on a ``β``-plane.
+#
+# # The "power method" for calculating growth rates and eigenmodes
+#
+# An "instability" is a small-amplitude solution that develops due to the presence
+# of an unstable "basic state" or background flow. Because these solutions are
+# small amplitude, we express any of the prognostic variables as
+#
+# ```math
+# u = û exp(σ t)
+# ```
+#
+# where ``σ`` is the "growth rate" of the instability. Our object is to
+# compute ``σ`` and also to get a feel for ``û``, which represents the 
+# "eigenmode", or the spatial structure of the instability.
```

Comment:
I don't quite agree with what you wrote here. May I rewrite?

Reply:
Yes please rewrite, this is just a place holder